crates/datafusion_ext/src/metrics.rs

```diff
@@ -223,22 +223,20 @@ impl AggregatedMetrics {
     pub fn new_from_plan(plan: &dyn ExecutionPlan) -> Self {
         let mut agg = AggregatedMetrics {
             elapsed_compute_ns: 0,
-            bytes_processed: 0,
+            bytes_read: 0,
         };
         agg.aggregate_recurse(plan);
         agg
     }
 
     fn aggregate_recurse(&mut self, plan: &dyn ExecutionPlan) {
-        let metrics = match plan.metrics() {
-            Some(metrics) => metrics,
-            None => return,
-        };
-        self.elapsed_compute_ns += metrics.elapsed_compute().unwrap_or_default() as u64;
-        self.bytes_processed += metrics
-            .sum_by_name(BYTES_PROCESSED_GAUGE_NAME)
-            .map(|m| m.as_usize() as u64)
-            .unwrap_or_default();
+        if let Some(metrics) = plan.metrics() {
+            self.elapsed_compute_ns += metrics.elapsed_compute().unwrap_or_default() as u64;
+            self.bytes_read += metrics
+                .sum_by_name(BYTES_PROCESSED_GAUGE_NAME)
```

Comment:
As Sean was saying, let's ujpdate the guage name

Reply:
Updated!